client/src/app/categories-page/categories-page.component.html

```diff
@@ -0,0 +1,140 @@
+<div class="container">
```

Comment:
This page should be split into components

Reply:
Ok